Gemfile

```diff
@@ -31,7 +30,6 @@ gem "roadie-rails"
 gem "sass-rails"
```

Comment:
I believe we can move this to assets as well.

Reply:
![Screenshot from 2020-07-23 02-23-18](https://user-images.githubusercontent.com/7680662/88227827-d9834280-cc8b-11ea-836a-590784d55d93.png)

I remember that we want to take this up in a follow-up but just want to point this out. sassc is 20% of our image size.

Reply:
I can take a look how to remove that as well.